src/ess/imaging/workflow.py

```diff
@@ -48,5 +74,8 @@ def YmirWorkflow() -> sl.Pipeline:
             HistogramModeDetectorsPath: DEFAULT_HISTOGRAM_PATH,
             ImageDetectorName: ImageDetectorName('orca'),
             RotationMotionSensorName: RotationMotionSensorName('motion_cabinet_2'),
+            BackgroundPixelThreshold: DEFAULT_BACKGROUND_THRESHOLD,
+            SamplePixelThreshold: DEFAULT_SAMPLE_THRESHOLD,
+            FileLock: DEFAULT_FILE_LOCK,
         },
     )
```

Comment:
> Tests will come from different PR

Unless this change is urgent (e.g. for the STAP demo), I would rather see the unit tests as part of the same PR?

Reply:
Yeah I wanted to release it before Christian leaves for vacation.
But I think I can just test them by myself.... I'll add tests here.